src/win/process.c

```diff
@@ -1251,6 +1251,18 @@ static int uv__kill(HANDLE process_handle, int signum) {
   }
 
   switch (signum) {
+    case SIGQUIT: {
+      MiniDumpWriteDump(
+        process_handle,
+        GetProcessId(process_handle),
+        MiniDumpNormal,
+        NULL,
+        NULL,
+        NULL,
+      );
```

Comment:
I am curious if this would work better (`MiniDumpWriteDump` claims to be single threaded):
```
HREPORT report;
WerReportCreate(L"SIGQUIT", WerReportNonCritical, NULL, &report)
WerReportAddDump(report, process_handle, NULL, WerDumpTypeMiniDump, NULL, 0);
WerReportSubmit(report, WerConsentDenied, 0, WER_SUBMIT_NO_ARCHIVE, NULL);
WerReportCloseHandle(report);
```
The user then may be able to enable it manually via:
https://learn.microsoft.com/en-us/windows/win32/wer/collecting-user-mode-dumps?source=recommendations

Note: completely untested code, just written by reading the docs and hoping they are accurate.

Reply:
Just to close the loop publicly, we spent quite a while trying to coerce WER to output dumps locally, but it just didn't work well at all.  We have decided instead to use `MiniDumpWriteDump()` in combination with a little bit of windows registry parsing to emulate similar behavior, so as to get local dumps written out to a stable location for both uncontrolled exits, as well as `SIGQUIT`'s.